docs/src/documentation/03-components/04-overlay/popover/02-react.mdx

```diff
@@ -2,6 +2,6 @@
 title: React
 ---
 
-import PopoverReadMe from "@kiwicom/orbit-components/src/Popover/README.md"
+import PopoverReadMe from "@kiwicom/orbit-components/src/Popover/README.md";
```

Comment:
Why only change this? It was the pattern everywhere to use `src` and it works fine for each page. (In part because of how files are included in the monorepo, so we don't need to source them from the published package.) If it's going to be changed, it should be changed everywhere.